backend/game_server/lib/helpers/vector.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Helpers.NotVector do defexception message: "One of arguments is not Vector" end
+
+
+
+defmodule Helpers.Vector do
+	alias :math, as: Math
+	alias Helpers.Vector, as: Vector
+	defstruct x: 0.0, y: 0.0
+
+
+	def distance(d1 = %Vector{},d2 = %Vector{}) do 
```

Comment:
if you really need shortcut - `v1` is better, because this letter have real meaning :P 
it always good to use longer (not too long) variable names that give our context meaning. there u use pattern matching and functions are pretty short, so `v1` should be enough.